### ISSUE TITLE
Should the site have a more "local" brand?

### ISSUE THREAD

**Issue by stpaultim** [stpaultim/MyFavoriteMeetingPlace#38]
We discussed naming it something like "Twin Cities Meeting Places", but decided that might be a bit limiting for the long term. 

I sometimes wonder if the current name isn't too generic and hurts with local awareness. I am curious about what others think? 

Does the current name work for you?

**Comment by ballPointPenguin** [comment]
It seems like branding for "Twin Cities" may be helpful for the website. But I think the software itself is more valuable if it stays agnostic and so can be used by someone in a different area.

**Comment by stpaultim** [comment]
Agreed that we should keep the technology geographically agnostic so that it can be used anywhere. 

The advantage of the current name, is that the site can also be used elsewhere. I can completely imagine folks in another community asking to use the same site to add data for their community. 

Another option, would be to clone the site and maintain separate installations for different communities, should that come to pass. Not sure if we should even be talking about this at this point.

**Comment by ballPointPenguin** [comment]
agreed. Cross that bridge when we come...

**Comment by stpaultim** [comment]
A volunteer registered www.twincitiesworkspaces.com as a potential domain.
